app/src/main/java/org/oppia/android/app/player/exploration/BottomSheetOptionsMenuItemClickListener.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.android.app.player.exploration
+
+/**
+ * Listener for executing the correct function based on the options clicked in the
+ * [BottomSheetOptionsMenu].
+ */
+interface BottomSheetOptionsMenuItemClickListener {
+  /**
+   * Function for passing the selected itemId
+   *
+   * @param itemId: Resource Id of the option selected
+   * @return [Unit]
+   */
+  fun handleOnOptionsItemSelected(itemId: Int)
```

Comment:
Perhaps use an enum instead of a resource ID to limit the possibilities of what can be passed to this function (limiting possibilities generally results in more error-tolerant and understandable code).